index.js

```diff
@@ -7,10 +7,9 @@ module.exports = {
   options: {
     babel: {
       loose: true,
-    }
-  },
-
-  isDevelopingAddon() {
-    return true;
+      plugins: [
+        ['ember-modules-api-polyfill', { blacklist: { '@ember/debug': ['assert', 'deprecate', 'warn']} }],
```

Comment:
You should not need to add this manually.  It is automatically added for you when using ember-cli-babel@6.6.2 or higher.